### PR TITLE
*: add a helper function to check existence and version of gogo's protobuf

### PR DESCRIPTION
**Pull request by zhexuany**
While I was trying to generate new proto code, I made a mistake. I use the latest gogo's protobuf rather than v0.5. I felt like we need make this rule into code. That is why I create this PR. I wish I could be the last person to make the same mistake.

### CODE REVIEW

**Comment by zhexuany** [comment]
@coocood @shenli PTAL

**Comment by coocood** [comment]
LGTM

File: _help.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+GOGO_ROOT=${GOPATH}/src/github.com/gogo/protobuf
```
Comment (shenli) [inline _help.sh]: Why name it `_help.sh`? Is `help.sh` OK?

Comment (zhexuany) [reply _help.sh]: it is general convention in bash. Utility stuff always starts with prefix `_`.

Comment (shenli) [reply _help.sh]: Got it!

**Comment by shenli** [comment]
LGTM

**Pull request merged**